admin_manual/configuration_server/index.rst

```diff
@@ -1,6 +1,6 @@
-=============================
-ownCloud Server Configuration
-=============================
+====================
+Server Configuration
```

Comment:
IMO ownCloud should be still specified, to avoid confusion with DB, Apache, etc.